apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -311,6 +311,8 @@ def from_h5ad(
           The default is to ingest them all. Use ``uns_keys=[]``
           to not ingest any ``uns`` keys.
 
+        additional_metadata: Optional metadata to add to the ``Experiment`` and all descendents.
```

Comment:
Let's emphasize that this is a blanket same-for-all convenience. And tell them that if they want to set metadata on individual elements they can do so using the SOMA API

```
with soma.open(uri, 'w') as exp:
    exp.metadata.update({"foo":"bar"})
    exp.obs.metadata.update({"foo":"whatever"})
    ...
```

Reply:
Good idea, lmk how the latest version looks.